opensearch/output.tf

```diff
@@ -1,30 +1,72 @@
 locals {
+  effective_audit_storage_type = var.opensearch_cluster.audit == null ? "" : (
+    var.opensearch_cluster.audit.external != null ?
+    "external_opensearch" :
+    "internal_opensearch"
+  )
```

Comment:
Looking at this, I think I would do an if else in the template instead and do away with this intermediate variable. Less cognitive overhead I believe.